shell/platform/fuchsia/flutter/accessibility_bridge_unittest.cc

```diff
@@ -22,11 +22,13 @@ namespace flutter_runner_test {
 class AccessibilityBridgeTestDelegate
     : public flutter_runner::AccessibilityBridge::Delegate {
  public:
+  float view_pixel_ratio = 1.f;
```

Comment:
For tests it would probably be helpful to set this to some other value so we can check the transforms are applied correctly.

Reply:
Yes, I make use of this float in ApplyViewPixelRatioToRoot test

Reply:
I see that it's used, but I think it should be some value other than 1.0, which doesn't result in a particularly interesting transformation right?